src/services/rooms.service.js

```
@@ -0,0 +1,64 @@
+import { Room } from '../models/room.js';
+
+const getAllRooms = async () => {
+  return Room.findAll({
+    order: [['createdAt', 'DESC']],
+  });
```

`getAllRooms` returns all rooms without pagination or error handling. At minimum, wrap calls in try/catch in controllers (or let the service throw and controllers map errors). If the dataset can grow, consider adding pagination parameters to avoid heavy responses.